### DISCUSSION TITLE
Roadmap / release of GitHub actions

### DISCUSSION THREAD

**Discussion by oldskool** [community/community#25103] [Actions]
<p>Is there currently an ETA on when GitHub Actions will come out of beta and into general availability?</p>
<p>Our company is currently looking at alternative ways for our CI/CD pipelines to run and I’ve been able to build succesful proof of concepts with GitHub Actions and am very excited about it. It’s ease of use and performance outweigh other alternatives I’ve tried. Although with it being still in beta and therefor no guarantees on stability and uptime, I cannot recommend a full migration just yet.</p>
<p>But if the release is just like weeks away, that is time we can probably wait out. So I’m wondering if there are any (rough) estimates out there for this.</p>

**Comment by chrispat** [comment, marked as answer]
<p>We announced at the beta launch in August that we would GA on November 13th at GitHub Universe</p>

**Comment by in8sudku** [comment]
<p>Is Github actions available for general use?</p>
